### PR TITLE
For prototype deployment

### PR DESCRIPTION
**Pull request by galletti94**
Changing `start` script and changing `prebuild/build` to `prestart`. Also disabling webpack host check to avoid `Invalid Host Header` error in browser

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,51 +50,26 @@
...
-    "css-loader": "^1.0.1",
-    "eslint": "^4.19.1",
-    "eslint-plugin-patternfly-react": "^0.2.0",
-    "file-loader": "^2.0.0",
-    "font-awesome": "4.7.x",
-    "html-webpack-plugin": "^3.2.0",
-    "mini-css-extract-plugin": "^0.4.5",
-    "node-sass": "4.8.x",
-    "optimize-css-assets-webpack-plugin": "^5.0.1",
-    "prettier": "^1.14.2",
```
Comment (jeff-phillips-18) [inline package.json]: why move the dev dependencies to runtime dependencies?

File: package.json [hunk 2]
```diff
@@ -30,51 +50,26 @@
...
+    "webpack": "^4.29.0",
     "webpack-bundle-analyzer": "^3.0.3",
     "webpack-cli": "^3.1.2",
-    "webpack-dev-server": "^3.1.10",
+    "webpack-dev-server": "^3.1.14",
     "webpack-merge": "^4.1.4"
   },
   "scripts": {
-    "prebuild": "yarn clean",
-    "build": "webpack --config webpack.prod.js",
```
Comment (jeff-phillips-18) [inline package.json]: we will need build if we want to run from dist in production mode which is what we ought to do (much smaller download size)

File: package.json [hunk 3]
```diff
@@ -30,51 +50,26 @@
...
-    "start": "webpack-dev-server --hot --color --progress --info=true --config webpack.dev.js",
-    "mock": "webpack-dev-server --hot --color --progress --info=true --config webpack.dev.js --define process.env.MOCK_MODE='\"true\"'",
```
Comment (jeff-phillips-18) [inline package.json]: We currently need mock mode to have a work around available for the github api throttle

File: package.json [hunk 4]
```diff
@@ -30,51 +50,26 @@
...
+    "prestart": "webpack --config webpack.prod.js",
+    "start": "webpack-dev-server --public 0.0.0.0:8080 --hot --color --progress --info=true --config webpack.dev.js",
```
Comment (jeff-phillips-18) [inline package.json]: hmmm.... this doesn't work for development.

**Comment by galletti94** [comment]
@jeff-phillips-18 I am using npm to install and run things on OpenShift so I am checking in the package-lock.json files. Lmk what you think

File: frontend/package.json [hunk 5]
```diff
@@ -64,13 +64,12 @@
     "webpack-merge": "^4.1.4"
   },
   "scripts": {
-    "prebuild": "yarn clean",
-    "build": "webpack --config webpack.prod.js",
+    "build": "webpack --config webpack.dev.js",
```
Comment (jeff-phillips-18) [inline frontend/package.json]: do we need to change this?

File: README.md [hunk 6]
```diff
@@ -10,24 +10,15 @@ able to run anywhere that Kubernetes can run.
...
 
 ```
-$ cd Frontend
-$ yarn
-$ yarn build
-
-#### Node Server
-cd server
-yarn
+npm run-script build
```
Comment (jeff-phillips-18) [inline README.md]: can we use `yarn` rather than `npm`?

File: frontend/package.json (hunk 5 above)
Comment (galletti94) [reply frontend/package.json]: Good question. It works with dev for sure. Need to check if it works with prod

File: README.md (hunk 6 above)
Comment (galletti94) [reply README.md]: why? I am having a hard time with yarn permissions on openshift...

Comment (jeff-phillips-18) [reply README.md]: just to be consistent, no big deal.

File: server/scripts/update-operators.sh [hunk 7]
```diff
@@ -1,10 +1,17 @@
 #!/bin/sh
+
```
Comment (jeff-phillips-18) [inline server/scripts/update-operators.sh]: Can we `pushd` up one level and to `data` or something? Having the `community-operators` directory under scripts doesn't seem reasonable.

Comment (galletti94) [reply server/scripts/update-operators.sh]: That makes much more sense - I'll add that now

File: build.sh [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+npm run-script build
+if [ ! -d ./dist/ ]; then
+    echo "Frontend build failed - no dist/ created"
+    exit 1
+fi
+echo "Frontend build successful"
+popd
+pushd server
+npm install
+pushd scripts
```
Comment (jeff-phillips-18) [inline build.sh]: Can we run `ln -s ../frontend/dist public` here? Some routing issues require us to access the dist directory without using a `../`

File: README.md (hunk 6 above)
Comment (jeff-phillips-18) [inline README.md]: OK with this but would be better if we could use `yarn` for consistency.

**Pull request merged**